server/controller/src/main/java/ai/starwhale/mlops/datastore/WalManager.java

```diff
@@ -427,6 +433,7 @@ private void writeToObjectStore(boolean clearOutput) {
                 inBuf.limit(this.outputStream.getOffset());
                 compressedSize = Snappy.compress(inBuf, outBuf);
             }
+            assert compressedSize <= outBuf.remaining();
```

Comment:
`assert` clauses will only be executed when running `java -ea xxx.jar`. If this is debug only, it's fine. If this is not for debugging, please add `-ea` to the jvm starting options in `docker/Dockerfile.server`  and catch the `AssertionError` excpeiton

Reply:
useless debug code, removed